src/AISmart.Application/Service/CqrsService.cs

```diff
@@ -24,4 +33,19 @@ public async Task SendEventCommandAsync(EventBase eventBase)
     {
         await _cqrsProvider.SendEventCommandAsync(eventBase);
     }
+
+    public async Task<K> QueryGEventAsync<T, K>(string index, string id) where T : GEventBase
+    {
+        try
+        {
+            var documentContent =  await _cqrsProvider.QueryGEventAsync(index, id);
```

Comment:
documentContent could be null?

Reply:
If there is no data, it will be null, and it will also be null after mapper If it is determined whether it is null, it is not supported to return null. For example
![Uploading image.png…]()

Reply:
Should we add a check if documentContent is null before we do deserializeObject?